de.dlr.sc.virsat.uiengine.ui/META-INF/MANIFEST.MF

```diff
@@ -39,6 +39,7 @@ Export-Package: de.dlr.sc.virsat.uieingine.ui,
  de.dlr.sc.virsat.uiengine.ui.editor,
  de.dlr.sc.virsat.uiengine.ui.editor.provider,
  de.dlr.sc.virsat.uiengine.ui.editor.snippets,
+ de.dlr.sc.virsat.uiengine.ui.editor.snippets.dialog,

```

Comment:
I think you're not using this dependenicy anymore, are you? Then we should remove it... It's always important to have the viewest depencies as possible

Reply:
I removed it.